openedx/core/djangoapps/user_api/views.py

```diff
@@ -65,24 +65,30 @@ def get(self, request):
 
         """
         form_desc = FormDescription("post", reverse("user_api_login_session"))
+        email_or_username = settings.FEATURES.get('ENABLE_LOGIN_BY_EMAIL_OR_USERNAME', False)
 
         # Translators: This label appears above a field on the login form
         # meant to hold the user's email address.
         email_label = _(u"Email")
+        if email_or_username:
+            email_label += _(u" or username")
 
         # Translators: This example email address is used as a placeholder in
         # a field on the login form meant to hold the user's email address.
         email_placeholder = _(u"username@domain.com")
 
         # Translators: These instructions appear on the login form, immediately
         # below a field meant to hold the user's email address.
-        email_instructions = _("The email address you used to register with {platform_name}").format(
+        email_instructions = _("The email address")
```

Comment:
`_( )` is `gettext` literals. So all such string (after parse this code with the special command) will be saved into special `po` files. I think because of this you can't split this line into separate words. The better way to create `if-else` expression with full sentence in case of 'True` and with the full sentence in case of 'False`